patches/minecraft/net/minecraft/item/SpawnEggItem.java.patch

```diff
@@ -0,0 +1,55 @@
+--- a/net/minecraft/item/SpawnEggItem.java
++++ b/net/minecraft/item/SpawnEggItem.java
+@@ -35,18 +35,28 @@
+ 
+ public class SpawnEggItem extends Item {
+    public static final Map<EntityType<?>, SpawnEggItem> field_195987_b = Maps.newIdentityHashMap();
++   public static final java.util.List<SpawnEggItem> MOD_EGGS = new java.util.ArrayList<>();
+    private final int field_195988_c;
+    private final int field_195989_d;
+-   private final EntityType<?> field_200890_d;
++   private final java.util.function.Supplier<? extends EntityType<?>> field_200890_d;
+ 
++   @Deprecated //FORGE: use constructor with Supplier parameter instead
```

Comment:
Please edit this line to match the future Omnibus PR. See https://github.com/MinecraftForge/MinecraftForge/pull/7612/commits/dae2fb7e20fdf46d1baee2ee2d9212d78daeb69b